src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisProvider.java

```diff
@@ -47,7 +47,7 @@ public OpenSecurityAnalysisProvider() {
                                                          LimitViolationFilter limitViolationFilter, ComputationManager computationManager,
                                                          SecurityAnalysisParameters securityAnalysisParameters, ContingenciesProvider contingenciesProvider,
                                                          List<SecurityAnalysisInterceptor> interceptors) {
-        OpenSecurityAnalysis osa = new OpenSecurityAnalysis(network, limitViolationDetector, limitViolationFilter, matrixFactory, connectivityProvider);
+        AbstractSecurityAnalysis osa = new AcSecurityAnalysis(network, limitViolationDetector, limitViolationFilter, matrixFactory, connectivityProvider);
```

Comment:
You shoud run either Ac or Dc, based on `lfParameters.isDc()`, like it is done in `OpenSensitivityAnalysisProvider`

Reply:
Yes this is what I did.